data_structures/double_linked_list.go

```diff
@@ -0,0 +1,58 @@
+package data_structures
+
+// import "fmt"
```

Comment:
Why is everything commented? If this file is not required, you can remove it @Kally95